provider/dynamodb.go

```diff
@@ -259,6 +259,25 @@ func (store *dynamodbOnlineStore) DeleteTable(feature, variant string) error {
 	return nil
 }
 
+func (store *dynamodbOnlineStore) Check() (bool, error) {
+	listOutput, err := store.client.ListTables(&dynamodb.ListTablesInput{Limit: aws.Int64(1)})
+	if err != nil {
+		return false, err
+	}
+	if len(listOutput.TableNames) == 0 {
+		return false, fmt.Errorf("(DynamoDB Health Check) no tables found")
+	}
+	scanInput := &dynamodb.ScanInput{TableName: listOutput.TableNames[0], Limit: aws.Int64(1)}
+	scanOutput, err := store.client.Scan(scanInput)
+	if err != nil {
+		return false, err
+	}
+	if len(scanOutput.Items) == 0 {
```

Comment:
what happens if the table is empty?

Reply:
That's a good point. I was looking for the `SELECT 1;` equivalent in DynamoDB and a couple of SO comments pointed to `Scan`, but perhaps I should only fail the health check if the `Scan` returns an error. Thanks for the attention to detail!